chamber/models/__init__.py

```diff
@@ -201,6 +232,7 @@ def save(self, *args, **kwargs):
                 self._save(*args, **kwargs)
         else:
             self._save(*args, **kwargs)
+        self.changed_fields = ChangedFields(self)
```

Comment:
všimněte si, že se inicializuje znovu až po post_save tedy když zavoláte self.changed_fields, tak to obsahuje správné změny ještě před save